architecture/adr-list/adr-008-ledger-resources.md

```diff
@@ -230,140 +190,217 @@ Example using the Veramo CLI:
 
 Example:
 
 ```jsonc
 {
-    "kms": "local",
-    "payload": { // example of resource header
-        "collectionId": "DAzMQo4MDMxCjgwM",
-        "id": "bb2118f3-5e55-4510-b420-33ef9e1726d2",
-        "name": "PassportSchema",
+    "payload": {
+        "data": "eyJhdHRyIjpbIm5hbWUiLCJhZ2UiXX0=",
+        "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
+        "id": "54cb8b4d-af33-4606-bc54-0f035ee30e0f",
+        "name": "PassportScheme",
+        "version": "1.0",
         "resourceType": "CL-Schema",
-        "data": "SGVsbG8sIHdvcmxk" // file with resource encoded into base64
+        "alsoKnownAs": [
+            {
+                "uri": "https://example.com/alternative-uri",
+                "description": ""
+            },
+            {
+                "uri": "https://example.com/alternative-uri",
+                "description": "Alternative URI description"
+            }
+        ]
     },
-    "signInputs": [{
-        "verificationMethodId": "did:cheqd:testnet:DAzMQo4MDMxCjgwM#key-1",
-        "keyType": "Ed25519",
-        "privateKeyHex": "0f5c124886178037952e87e0cdc55d185732577fca19ae877e64ac9ab24a0cc534e5326e70f1a42d785d93048aee806c359ec75a7b06f39253befd1746708438"
-    }]
+    "signInputs": [
+        {
+            "verificationMethodId": "did:cheqd:testnet:91e5f0cf-5f1e-5c19-97d3-d313e84033b4#key-1",
+            "privKey": "tBjxEJCqSkj7u+iTWRqAVZwtcl2XBZrlaMfhxYIRc4wj7epbmDkJ35sCin3MWnAxvHJNDY0yyPafVspsrgb1Ng=="
+        }
+    ]
 }
 ```
 
 #### MsgCreateResourceResponse
 
-* Resource: [Resource](#resource)
+* Returns created `Resource`.
 
 Example:
 
 ```jsonc
-{ "resource":  <Resource> }
+{
+    "resource": {
+        "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
+        "id": "54cb8b4d-af33-4606-bc54-0f035ee30e0f",
+        "name": "PassportScheme",
+        "version": "1.0",
+        "resourceType": "CL-Schema",
+        "alsoKnownAs": [
+            {
+                "uri": "https://example.com/alternative-uri",
+                "description": ""
+            },
+            {
+                "uri": "https://example.com/alternative-uri",
+                "description": "Alternative URI description"
+            }
+        ]
+    }
+}
 ```
 
-#### QueryGetCollectionResourcesRequest
+#### QueryCollectionResourcesRequest
 
-* Collection ID: String - an identifier of linked DIDDoc
+* `Collection ID`: String - an identifier of linked DIDDoc.
   
 Example:
 
 ```jsonc
-{ "collectionId": "DAzMQo4MDMxCjgwM" }
+{ "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4" }
 ```
 
-#### QueryGetCollectionResourcesResponse
+#### QueryCollectionResourcesResponse
 
-* Resources: [ResourceHeader\[\]](#resourceheader)
+* Returns `collection of resources` created by the specific `collection ID`.
 
 Example:
 
 ```jsonc
-{ "resources":  [<ResourceHeader1>, <ResourceHeader2>] }
+{
+    "resources": [
+        {
+            "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
```

Comment:
Could you also check do we use snake case for resource collection call also?

Reply:
Done! ✅